src/main.rs

```diff
@@ -1,12 +1,9 @@
-use tokio;
-use bson::{doc, Bson};
-use chrono::TimeZone;
-use chrono::Utc;
-use mongodb;
+use chrono::{TimeZone, Utc};
+use mongodb::bson::{self, doc, Bson};
```

Comment:
No need to `use mongodb;` or `use serde_json;` dependency crates already are in scope. I also merged/alphabetized the imports, which I tend to prefer for readability, but this isn't strictly necessary.